core/templates/dev/head/pages/exploration_editor/statistics_tab/issues_directive.html

```diff
@@ -0,0 +1,28 @@
+<div class="oppia-issues-issue-dir">
+  <div class="row oppia-issues-content oppia-issues-row-eq-height">
+    <div class="col-md-2 oppia-issues-border-gray oppia-ft-14">
+      <div ng-repeat="issue in issues" class="row">
+        <div ng-if="getIssueIndex(issue) == 0" class="issueNav text-center col-md oppia-issues-bor-btm oppia-issues-clr-green oppia-pt2 oppia-pb2 oppia-issues-bg-clr oppia-issues-text-white" ng-click="makeVisible(getIssueIndex(issue))" id="issueNav<[getIssueIndex(issue)]>">
+          Issue <[getIssueIndex(issue) + 1]>
+        </div>
+        <div ng-if="getIssueIndex(issue) !== 0" class="issueNav text-center col-md oppia-issues-bor-btm oppia-issues-clr-green oppia-pt2 oppia-pb2" ng-click="makeVisible(getIssueIndex(issue))" id="issueNav<[getIssueIndex(issue)]>">
+          Issue <[getIssueIndex(issue) + 1]>
```

Comment:
Can we use: `item in items track by $index` instead here of calculating the issue index?

It's similar to Python's `enumerate` function.

Reply:
@brianrodri I tried this first, doesnt seem like it works. Kept throwing syntax error. Thats why I moved to the helper function for identifying indices.

Reply:
OK, let's meet up and figure out why.